docker/generic/build.sh

```diff
@@ -78,6 +79,7 @@ BASE=$IMAGE_NAME:$TAG_PREFIX-$ROS_DISTRO-base
 docker build \
     --tag $BASE \
     --build-arg ROS_DISTRO=$ROS_DISTRO \
+    --build-arg ROS2_DISTRO=${ROS2_DISTRO} \
```

Comment:
You may drop the curly brackets syntax to be consistent with the rest of the file.

Reply:
Yep